README.md

```diff
@@ -172,8 +192,12 @@ a migration of a critical backend system.
 
 [Sentry](http://blog.getsentry.com/2015/07/23/transaction-id-wraparound-in-postgres.html). Transaction ID Wraparound in Postgres caused Sentry to go down for most of a working day.
 
+[Skyliner](https://blog.skyliner.io/post-mortem-outages-on-1-19-17-and-1-23-17-3f65cc6f693e). A memory leak in a third party library lead to Skyliner being unavailable for 4 days(!).
```

Comment:
Reading the linked post, it sounded more like the 2 incidents, while having the same root cause, were separate instances with a 4 day gap of regular service in between.